### PR TITLE
removed deprecated cli flags

### PR DESCRIPTION
**Pull request by acud**

### CODE REVIEW

File: cmd/swarm/config.go
Comment (nonsense) [inline cmd/swarm/config.go]: Maybe remove this altogether?

Comment (acud) [reply cmd/swarm/config.go]: i thought about it but also realised we would probably again deprecate stuff so I thought to just leave the infrastructure just in case...

**Review by nonsense — APPROVED** [review]
LGTM, but probably best to wait for a second review prior to merging.

File: cmd/swarm/config.go
Comment (acud) [reply cmd/swarm/config.go]: agree. removing

**Comment by nonsense** [comment]
@justelad once this is merged, we should update `swarm-cluster` as there is a reference to `ens-addr` at three places.

**Comment by acud** [comment]
@nonsense 👍

**Review by nolash — APPROVED** [review]

**Pull request merged**
